src/azure-cli/azure/cli/command_modules/apim/tests/latest/test_apim_scenario.py

```diff
@@ -87,8 +87,56 @@ def test_apim_core_service(self, resource_group, resource_group_location, storag
             self.check('provisioningState', 'Succeeded')
         ])
 
-        # delete command
-
+        # api operations
```

Comment:
Is there any problem with these tests ?

Reply:
In src/azure-cli/azure/cli/command_modules/apim/tests/latest/test_apim_scenario.py:109:
ERROR    cli.azure.cli.core.parser:parser.py:153 az: error: unrecognized arguments: API True

I don't know whats wrong.

Reply:
Let's just add these tests and make sure they can pass.

Reply:
So I've figured out how to run tests locally and successfully executed a test run. But now somehow Azure DevOps doesn't like the output:

```
except CannotOverwriteExistingCassetteException as ex:
>           raise AssertionError(ex)
E           AssertionError: No match for the request (<Request (PUT) https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/%7Brg%7D/providers/Microsoft.ApiManagement/service/cli-test-apim-000003/apis/az-cli000004?api-version=2019-01-01>) was found. 
```

Obviously he didn't take the {rg} parameter properly:
`resourceGroups/%7Brg%7D/providers`

with following command:
`command = 'apim api create -g "{rg}" --service-name "cli-test-apim-000003" --display-name "Contoso API" --path "test" --api-id "az-cli000004" --protocols "https" --service-url "https://contoso.com" --tags "foo=baz"'`

Reply:
Okay I've added a rg kwarg variable and now it works :)